### ISSUE TITLE
filter for specific attester organization on UI

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/DeVouch-FE#177]
Showing to the user when we have applied filtering for a specific attester group that the user knows they are sorting by most vouches from given attester organizations.
![Image](https://github.com/user-attachments/assets/f7edd893-c5bb-473d-b109-850b1205fe6e)

## AC 
on the all projects page:
- [x] when filtering by source platform the filter selection component is shown above list of projects 
- [x] when filtering by attester org the filter selection component is shown above list of projects 
- [x] the filtered item selected from drop down filter menu matches the one shown in the component text
- [x] can remove a filter by clicking the `x` on the filter component
- [x] removing the filter from the drop down menu also removes the corresponding filter component

**Comment by Tosinolawale** [comment]
![Filter](https://github.com/user-attachments/assets/3b4d5840-048f-49c2-abd1-fef79d140269)
https://www.figma.com/design/QBvhJZY3C93a8NKqcMnEHX/DeVouch?node-id=8310-13429&t=dIG7jJEkfjqgr5Oe-1
@divine-comedian @mateodaza @MohammadPCh

**Comment by mateodaza** [comment]
Linking this [issue](https://github.com/Giveth/DeVouch-BE/issues/118) here as reference from the backend

**Comment by divine-comedian** [comment]
@Tosinolawale what does this design look like if multiple sources and groups are filtered for?  How does it handle overflow if we fill up the whole line block with filter conditions? 
i.e. Imagine the possible filter conditions 

> **Attested By**: Gitcoin Passport Holders, Optimism Badgeholders, Giveth Verifiers **From:** Retro Funding 4, Retro Funding 5, Gitcoin, Giveth

**Comment by mateodaza** [comment]
@Tosinolawale does this design make sense if we add more than one organization on the filter?

**Comment by Tosinolawale** [comment]
> @Tosinolawale what does this design look like if multiple sources and groups are filtered for? How does it handle overflow if we fill up the whole line block with filter conditions? i.e. Imagine the possible filter conditions
> 
> > **Attested By**: Gitcoin Passport Holders, Optimism Badgeholders, Giveth Verifiers **From:** Retro Funding 4, Retro Funding 5, Gitcoin, Giveth

@divine-comedian @mateodaza that Page itself is a component variant which detailed the multiple selection states  and overflow. I should have shown the different states clearly.  
I have shown the separate states here 
![Filter](https://github.com/user-attachments/assets/9228c49f-0e38-4942-8612-758349dbef64)
https://www.figma.com/design/QBvhJZY3C93a8NKqcMnEHX/DeVouch?node-id=8310-13429&t=IKoPfB4QtGGSnX17-1

**Comment by divine-comedian** [comment]
@mateodaza - I think we pushed this out too with the latest changes to sorting / filtering - can you confirm if its ready for QA?

**Comment by mateodaza** [comment]
> @mateodaza - I think we pushed this out too with the latest changes to sorting / filtering - can you confirm if its ready for QA?

Yep I think @LatifatAbdullahi already did it. If not, it's ready for QA

**Comment by LatifatAbdullahi** [comment]
- When filtering by source platform the filter selection component is shown above list of projects - **Pass**
 -When filtering by attester org the filter selection component is shown above list of projects - **Pass**
 -The filtered item selected from drop down filter menu matches the one shown in the component text - **Pass**
- Removing a filter by clicking the x on the filter component - **Pass**
- Removing the filter from the drop down menu also removes the corresponding filter component - **Pass**



@mateodaza 

**UI issues**

**Staging**

![dhj](https://github.com/user-attachments/assets/a5963824-4a63-4b77-88bf-e9c72e40a4f5)



**Design**

![k](https://github.com/user-attachments/assets/3ac90663-2c45-4582-9160-f67a0cb0d854)

![uuu](https://github.com/user-attachments/assets/36fee0c0-a465-4eeb-935f-a23d10ccb698)


- In the "Sort by" section the font weight of the selected option should be **700**
- The background of the Filter selection component should not be white, please check the design above
- The Filter selection component in the Design, there is no colon (please remove the colon)
- In the Design there is no search bar  component @Tosinolawale  @divine-comedian (**Are we removing the Search bar component?**)
- From the design, when Users select 3 or more Source platforms to be filtered, we should have atmost 3 filter tags in a single line, same goes for filtering Attested Org (Currently, if we have 3 or more tags, only 2 tags are in a single line instead of 3)

**Comment by divine-comedian** [comment]
The search bar is definitely staying on that page 

@Tosinolawale can you confirm the position of the search bar is correct? 
@mateodaza I'll let you take a look and correct the other UI issues reported.

**Comment by Tosinolawale** [comment]
> The search bar is definitely staying on that page
> 
> @Tosinolawale can you confirm the position of the search bar is correct? @mateodaza I'll let you take a look and correct the other UI issues reported.
![Component 1](https://github.com/user-attachments/assets/e7a0af02-828b-406d-9edf-c8d71663a77d)
@mateodaza @LatifatAbdullahi

**Comment by divine-comedian** [comment]
tested this out on staging, all AC passes, to note when you select multiple attester orgs it works like an OR filter

meaning it will check the project and show it if it has at least one attestation from any of the specified attester orgs
